config/toml.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+const DefaultConfigTemplate = `# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Base Configuration                            ###
+###############################################################################
+
+log-level = "{{ .BaseConfig.LogLevel }}"
+oracle-mnemonic = "{{ .BaseConfig.OracleMnemonic }}"
+data-encryption-key-file = "{{ .BaseConfig.DataEncryptionKeyFile }}"
+
+###############################################################################
+###                         Panacea Configuration                           ###
+###############################################################################
+
+[panacea]
+
+grpc-addr = "{{ .Panacea.GRPCAddr }}"
+
+###############################################################################
+###                         Enclave Configuration                           ###
+###############################################################################
+
+[enclave]
+
+enable = {{ .Enclave.Enable }}
+attestation-provider-addr = "{{ .Enclave.AttestationProviderAddr }}"
```

Comment:
We don't need this `attestation-provider-addr` anymore for doracles. We are gonna use the Azure DCAP client that is used automatically by `enclave.VerifyReport(...)`. I've already installed all libraries for Azure DCAP client on our Azure VMs.
https://github.com/medibloc/doracle-poc/blob/dd4bacaaa029e7217872279ad87b19d4412f48f0/README.md?plain=1#L20